src/app/admin/admin-asset-new/admin-asset-new.component.html

```diff
@@ -0,0 +1,142 @@
+<div class="w-full h-full flex justify-center mt-8 px-2 pb-8">
+  <div class="w-full max-w-6xl">
+    <div class="flex flex-col gap-2 w-full justify-start bg-white shadow-xl rounded-4xl p-4 pt-8">
+      <form [formGroup]="createForm">
+        <div class="text-2xl text-indigo-900 mb-8">Set up your asset and its related token</div>
+        <div class="flex flex-col lg:flex-row">
+          <div class="relative w-full lg:w-1/3">
+            <div class="flex justify-center w-full absolute top-32 lg:top-40">
+              <label class="flex items-center gap-4 px-8 py-2 rounded-full bg-green-600 text-white text-lg
+                              cursor-pointer">
+                <input class="hidden" type="file" formControlName="logo" multiple>
+                <svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path d="M6 1V6M6 6V11M6 6H11M6 6H1" stroke="white" stroke-width="2" stroke-linecap="round"
+                        stroke-linejoin="round"/>
+                </svg>
+                <span>Upload logo</span>
+              </label>
```

Comment:
can we stop using `cursor-pointer` for clickable items? I've mentioned it before why this is the issue. Either button or anchor tag. 
Is there maybe an issue with this approach?

Reply:
I had to use it here since I need to use the `label` tag to enable file upload. I tried both wrapping it inside `button` and `a` tags and it doesn't work. Also, putting `button` or `a` inside `label` tag did not work. So using `cursor-pointer` is the simplest workaround with this one.